### ISSUE TITLE
NTR: Aldolase A activity low in RBC (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8223]
New term request Aldolase A activity low in RBC (segal_120302195214): Fructose-1,6-bisphosphate aldolase; red blood cells, erythrocytes

**Comment by pnrobinson** [comment]
@MickeySegal
I am assuming this is
 Glycogen storage disease XII  and thus we mean Fructose-1,6-bisphosphate aldolase ([EC 4.1.2.13](https://enzyme.expasy.org/EC/4.1.2.13))

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000555
- label: Decreased erythrocyte fructose-1,6-bisphosphate aldolase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:2825199
- def: Activity or concentration of fructose-1,6-bisphosphate aldolase (EC 4.1.2.13) in red blood cells below the lower limit of normal.
- comment: Fructose-1,6-bisphosphate aldolase A (fructose-bisphosphate aldolase; EC 4.1.2.13) deficiency is an autosomal recessive disorder associated with hereditary hemolytic anemia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8223
- synonyms:
